charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,93 @@
+
+# Expression Matrix Service
```

Comment:
Although we have been loose about the difference between counts and expression matrices, it may be good to tighten that up here. Is this an expression matrix service, a counts matrix service or both? If expression, this assumes UMI collapsing, correction for depth (CPM) and/or transcript length (TPM), I believe we should always deliver counts and optionally deliver other things.

Reply:
I don't see the "expression" term as overloaded the way you do. I think of an expression matrix as the general term, and count matrices, cpm matrices and tpm matrices are the specific types. 

In any case, I believe we're aligned on delivery of the raw counts, augmented with transcript lengths where appropriate.

Reply:
Maybe just using the term "matrix service" for the charter and not specifying the data type will get around this and make this service to be positioned to be applying for many data modalities long-term.

Reply:
Interesting idea. The co-delivery of those objects always been my plan.Because the idea and core deliverables remain the same, I don't think there is need for change. 

As a thought experiment, If something not related to expression were to be made into a matrix, I don't think this service would produce it.

Reply:
Works for me!